### PR TITLE
Cleanup cleanup-expose

### PR DESCRIPTION
**Pull request by cmsj**

### CODE REVIEW

File: extensions/expose/init.lua [hunk 1]
```diff
@@ -318,11 +321,11 @@ local function getHints(self,windows)
     end
   end
   local function normalize(t,n) --change in place
-    local _
     while #t>0 and tlen(t.apps)>0 do
       if n>self.ui.maxHintLetters or (tlen(t.apps)==1 and n>1 and not hasSubHints(t))  then
         -- last app remaining for this hint; give it digits
-        local app=next(t.apps)
```
Comment (cmsj) [inline extensions/expose/init.lua]: I haven't checked this in detail yet, but I'm wondering if the `next()` call is important here even if `app` isn't used?

Comment (latenitefilms) [reply extensions/expose/init.lua]: @cmsj - Good point - that makes sense. It should probably just be `next(t.apps)` without the `local app`?

Comment (cmsj) [reply extensions/expose/init.lua]: That would be my expectation, yes.

Comment (latenitefilms) [reply extensions/expose/init.lua]: Fixed in my branch.

**Comment by cmsj** [comment]
Could you re-post your branch as a PR?

**Pull request closed without merging**